client/my-sites/upgrades/components/form/hidden-input.jsx

```diff
@@ -37,14 +35,14 @@ module.exports = React.createClass( {
 	render: function() {
 		if ( this.state.toggled ) {
 			return (
-				<Input { ...this.props } initialFocus={ this.state.focus } />
-			);
-		} else {
-			return (
-				<div className="hidden-input">
-					<a href="" onClick={ this.handleClick }>{ this.props.text }</a>
-				</div>
+				<Input { ...this.props } autofocus />
```

Comment:
Ah nm I'm wrong here. I was confused by this https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/upgrades/components/form/input.jsx#L49L53 ; Nice work on removing the focus mixin.